libbeat/docker-compose.yml

```diff
@@ -5,25 +5,31 @@ services:
     depends_on:
       - proxy_dep
     environment:
+      - LIBBEAT_PATH=/go/src/github.com/elastic/beats/libbeat
+      - BEAT_STRICT_PERMS=false
       - REDIS_HOST=redis
       - REDIS_PORT=6379
       - SREDIS_HOST=sredis
       - SREDIS_PORT=6380
       - LS_HOST=logstash
       - LS_TCP_PORT=5044
       - LS_TLS_PORT=5055
-      # Setup work environment
-      - LIBBEAT_PATH=/go/src/github.com/elastic/beats/libbeat
       - KAFKA_HOST=kafka
       - KAFKA_PORT=9092
       - KIBANA_HOST=kibana
       - KIBANA_PORT=5601
+      - KIBANA_USER=beats
+      - KIBANA_PASS=testing
+      - ES_HOST=elasticsearch
+      - ES_PORT=9200
+      - ES_USER=beats
+      - ES_PASS=testing
       - ES_MONITORING_HOST=elasticsearch_monitoring
       - ES_MONITORING_PORT=9200
       - ES_HOST_SSL=elasticsearchssl
       - ES_PORT_SSL=9200
-    env_file:
-      - ${PWD}/build/test.env
```

Comment:
Only heartbeat `docker-compose.yml` uses `test.env` after this change. WDYT about removing this also from there and then remove the `make write-environment` target too?

Reply:
Good catch. I missed it because the magefile.go didn't actually run the integration tests. I updated the magefile to run the integration tests and removed this from heartbeat.

I won't change the Makefile now. It won't hurt to leave it there for a bit. Once we have fully converted to mage then we can nuke a large portion of the Makefile at once. By fully converted I mean - updated Jenkins, Travis, APM, and also validate the generators are solid.